system/data/es/systems/accelerated.json

```diff
@@ -0,0 +1,166 @@
+{
+    "name": "Acelerado",
+    "identifier": "fae",
+    "types": {
+        "Aspects": [
+            {
+                "label": "Basico",
+                "entries": [
+                    {
+                        "name": "Concepto principal",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Concepto principal"
+                        }
+                    },
+                    {
+                        "name": "Complicación",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Complicación"
+                        }
+                    },
+                    {
+                        "name": "Otro aspecto",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Otro"
+                        }
+                    },
+                    {
+                        "name": "Otro aspecto",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Otro"
+                        }
+                    },
+                    {
+                        "name": "Otro aspecto",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Otro"
+                        }
+                    }
+                ]
+            },
+            {
+                "label": "Opcional",
+                "entries": [
+                    {
+                        "name": "Relación",
+                        "type": "aspect",
+                        "data": {
+                            "label": "Relación"
+                        }
+                    },
+                    {
+                        "name": "Aspecto vacio",
+                        "type": "aspect"
+                    }
+                ]
+            }
+        ],
+        "Consequences": [
+            {
+                "entries": [
+                    {
+                        "name": "Consecuencia leve",
+                        "type": "consequence",
+                        "data": {
+                            "label": "Leve",
+                            "icon": "2"
+                        }
+                    },
+                    {
+                        "name": "Segunda consecuencia leve (Condición de habilidad)",
```

Comment:
Personally, I'd remove the capital C from Condición and use lowercase instead (Spanish does not usually capitalize these kind of nouns).